diff --git a/node_modules/@config-plugins/react-native-branch/android/build.gradle b/node_modules/@config-plugins/react-native-branch/android/build.gradle
index ed48d73..0b508bf 100644
--- a/node_modules/@config-plugins/react-native-branch/android/build.gradle
+++ b/node_modules/@config-plugins/react-native-branch/android/build.gradle
@@ -11,7 +11,7 @@ useDefaultAndroidSdkVersions()
 useExpoPublishing()
 
 android {
-  namespace "expo.modules.haptics"
+  namespace "expo.modules.adapters.branch"
   defaultConfig {
     versionCode 5
     versionName '5.0.0'
